packages/api-explorer/src/JsonForm/get-custom-editor.js

```diff
@@ -9,6 +9,9 @@ const getCustomEditor = key =>{
       this._super(this.container)
     },
     build() {
+      if (key === 'multiple') {
```

Comment:
Why? Could we comment this?

Reply:
@Lucascanna can you please elaborate on this so that I can add a comment?

Reply:
done